src/main/java/com/spotify/docker/BuildMojo.java

```diff
@@ -118,6 +119,22 @@
   @Parameter(property = "dockerCmd")
   private String cmd;
 
+  /** The workdir for the image. Ignored if dockerDirectory is set */
+  @Parameter(property = "workdir")
+  private String workdir;
+
+  /** The user for the image. Ignored if dockerDirectory is set */
+  @Parameter(property = "user")
+  private String user;
+
+  /** 
+   * The run commands for the image. 
+   *  */
+  @Parameter(property = "dockerRuns")
```

Comment:
Awesome! I was going to add that. One question I have: is it guaranteed that the order in the `pom.xml` file will be the order received here?

If not it might be a problem